lesson10/public/app.js

```diff
@@ -0,0 +1,51 @@
+$('#submit').click(()=>{
+  const name = $('#name').val();
+  const comment = $('#comment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/nameCom',
+    data:JSON.stringify ({name, comment}),
```

Comment:
in the future start writing your objects like this: 
JSON.stringify({
name,
comment
});

Reply:
ok! I thought this way would be more efficient/save more lines of code